package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "faunadb",
-  "version": "2.14.2",
+  "version": "0.0.0-development",
```

Comment:
`semantic-release` changes the version in our `package.json` but handles updating the version when publishing based on the most recent commit tag/our current version on NPM

Reply:
I'm going to read the docs a bit more and see if we can keep our current version or if that **must** change. Don't want to cause any unforeseen publishing issues 😓

Reply:
@ryancharris When we add the same thing for console, we were able to keep the version in package.json an let the tool handle it. we did not have issues so far, maybe you can try to let the version field as it is on package.json, and generate a version locally using the --dry-run flag and see if it picks the correct number.

Reply:
@fauna-rodrigo Good call. I'll try that 👍 My understanding was that we could keep the version and `semantic-release` would be smart enough to bump the number appropriately based on the commit tag of the most recent commit on master.